client/components/fields/editor/EventRelatedPlannings/EmbeddedCoverageForm.tsx

```diff
@@ -174,6 +189,8 @@ export class EmbeddedCoverageFormComponent extends React.PureComponent<IProps> {
                             style={{padding: '2rem 0'}}
                         >
                             <SelectUser
+                                key={this.state.selectedDeskId}
```

Comment:
Use `selectedUserId` prop here instead of `key`. Key could be a solution if no proper way was available.

Reply:
Yep, I get it. Add `selectedDeskId` to state and set to `null` inside `onDeskChange` and you'll achieve it.

Reply:
No need, it's working as intended now. Besides this if I set it to `null` it counts as an invalid key. I'm using it as a key with `SelectUser`

Reply:
I'm not suggesting you to pass `selectedUserId` as a key, but an actual prop of `SelectUser` component.

Reply:
Got it, well I'm passing it but if it changes the `SelectUser` doesn't update based on it. Should I make the `SelectUser` keep it in its state so it updates. So far nothing else has worked so I can try this

Reply:
I've pushed a fix to client-core which fixes this issue. Now I don't need to pass a key and `SelectUser` handles updates internally